python/pyspark/pandas/frame.py

```diff
@@ -5500,6 +5500,25 @@ def op(psser: ps.Series) -> ps.Series:
         else:
             return psdf
 
+    def interpolate(
```

Comment:
It should be listed in https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.pandas/frame.rst to document.